14-truck_backer-upper.ipynb

```diff
@@ -249,7 +256,7 @@
    "source": [
     "# Build expert data set\n",
     "\n",
-    "episodes = 10\n",
+    "episodes = 100\n",
```

Comment:
Why changing this? Its only purpose it to show how data is generated.
To generate the training data set you want to use 10,000 episodes.